static/js/tools/map/compute/map_value_dates.ts

```diff
@@ -81,18 +71,17 @@ export function useComputeMapValueAndDate(
       chartStore.denomStat.data ? chartStore.denomStat.data.facets : {},
       chartStore.allStat.data ? chartStore.allStat.data.facets : {}
     );
-    for (const geoFeature of chartStore.geoJson.data.features) {
-      const placeDcid = geoFeature.properties.geoDcid;
-      let wantedFacetData = chartStore.defaultStat.data.data;
-      if (statVar.value.metahash) {
-        wantedFacetData = filterAllFacetData(
-          chartStore.allStat.data.data,
-          statVar.value.metahash
-        );
-        if (_.isEmpty(wantedFacetData)) {
-          continue;
-        }
+    let wantedFacetData = chartStore.defaultStat.data.data;
+    if (statVar.value.metahash) {
+      wantedFacetData = filterAllFacetData(
+        chartStore.allStat.data.data,
+        statVar.value.metahash
+      );
+      if (_.isEmpty(wantedFacetData)) {
```

Comment:
should this be outside the `if (statVar.value.metahash)`?

Reply:
chartStore.allStat.data.data should not be empty at this time with ready check, but good to move outside.